help/md/dnSBBDP.md

```diff
@@ -1,9 +1,34 @@
 ## name
 dnSBBDP
 ## title
+Serially sampled birth-death process
 ## description
+Simulates a tree under a birth-death process with a specified sampling rate.
 ## details
+dnSBBDP simulates a tree under a birth-death process without a character dependent effect.
+Additionally, the sampling probability of taxa can be specified using the rho argument of the 
+function, allowing for a specfic sampling rate of extinct taxon.
+
+This function is an older version of a birth death process with an explicit sampling rate.
+To see a newer version with more funcitonality see: dnBDSTP
```

Comment:
Typo on line 13: "funcitonality" should read "functionality". More importantly, though, it should be specified what this additional functionality is. Does `dnBDSTP` reduce to `dnSBBDP` if you fix some of its parameters? If so, which ones?